src/pages/user/UserRoles.tsx

```diff
@@ -1,9 +1,16 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Typography } from '@material-ui/core';
```

Comment:
litt vel raus med tomme linjer her :)

Reply:
har begynt å bruke import sorter igjen for å få det litt ryddigere 😛  men kan fjerne den configen da